### ISSUE TITLE
Need to take down current prod site and replace with a landing page

### ISSUE THREAD

**Issue by SophAm1** [hackforla/311-data#1441]
### Overview
We need to take down the current product site and replace it with a landing page. Hopefully the landing page can provide context about the site being under construction and meet the needs of the stakeholders. Also, there is a need to see if a url direct will have an impact on the infra. 

### Action Items
- [x] Create a roadmap for the product landing page
- [x] Create temporary landing page for prod 
- [x] Communicate with engineering to see if a url redirect may impact the infra
- [x] Get information about the necessary stakeholders and their intent
- [x] Create a simple landing page for the site page including email collection

### Resources/Instructions
https://myla311.lacity.org/ (for users to be redirected if they need to place a 311 ticket)

**Comment by edwinjue** [comment]
Hi All,

Here is a draft of the landing page. The verbiage is as per Bonnie's message. Please take a look and let me know if any changes need to be made before I update the `master` branch:

Desktop:
![image](https://user-images.githubusercontent.com/1448719/215656115-4c53d085-6ddd-4fc3-b444-d36514044f22.png)


Mobile:
![image](https://user-images.githubusercontent.com/1448719/215656030-7d945381-1866-4fa5-b82b-66a8c3fe501d.png)

**Comment by EchoProject** [comment]
@edwinjue looks great! I think we can simplify the wording to: 

"311-Data.com is temporarily unavailable while we prepare for our new version. Please check back again for our improved analytics tools. Thank you!"

**Comment by edwinjue** [comment]
@EchoProject here are screenshots of the landing page with the updated wording. 

Please let me know if this looks good to deploy so @bryantran04 @hqasmei and I can coordinate a time to work together to push these changes into production and when.

Also let us know if there is any other communication that needs to be coordinated and to whom before we do so. Thanks!

Desktop:
![image](https://user-images.githubusercontent.com/1448719/215879002-8b4b04a1-7aef-442c-84c4-90c50b6c48e2.png)

Mobile:
![image](https://user-images.githubusercontent.com/1448719/215879131-07e5c614-dead-47f0-972e-8a830706eb29.png)

**Comment by edwinjue** [comment]
Hi All, with the help of @bryantran04 and @hqasmei the landing page has been pushed into production. Please take a look and let us know if you have any questions or concerns.

**Comment by ExperimentsInHonesty** [comment]
Why did the text about redirecting people to https://myla311.lacity.org/ (for users to be redirected if they need to place a 311 ticket) get removed??!!!!  Please add that back in.

**Comment by pras4u** [comment]
Hi @ExperimentsInHonesty The team just implemented the change with the link to the City of LA's 311 site. Request you to review & provide your feedback. 
![image](https://user-images.githubusercontent.com/44175476/217718086-4c53c9be-8494-4e3b-a318-cbed685f272d.png)
